PowerEditor/src/NppDarkMode.cpp

```diff
@@ -2760,8 +2865,16 @@ namespace NppDarkMode
 
 	void setDarkTitleBar(HWND hwnd)
 	{
-		NppDarkMode::allowDarkModeForWindow(hwnd, NppDarkMode::isEnabled());
-		NppDarkMode::setTitleBarThemeColor(hwnd);
+		if (NppDarkMode::isWindows10(19041U))
```

Comment:
...
And use `if (NppDarkMode::isWindows11(19041U))` here instead?
(part 2/2)

Reply:
And I would use `if (NppDarkMode::isWindows10() && NppDarkMode::g_buildNumber >= 19041U))` (somethings like that) instead.

Reply:
@donho 
done